src/index.js

```diff
@@ -1,26 +1,32 @@
 const BUTTONS_ARRAY = [
     {
         value: 1,
+        inCallButtons: "mute"
```

Comment:
If I was you, I would create sepate array with in-call buttons AND should rename `BUTTONS_ARRAY` to highlight the fact that these buttons are for prefore call.